admin/controllers/administrator.js

```diff
@@ -6,6 +6,10 @@ const settingsErrorMessages = require('../lib/errors/settings')
 const settingsValidator = require('../lib/validator/settings-validator')
 const checkWindowValidator = require('../lib/validator/check-window-validator')
 const checkWindowErrorMessages = require('../lib/errors/check-window')
+const checkWindowService = require('../services/check-window.service')
+const dateService = require('../services/date.service')
+const checkWindowDataService = require('../services/data-access/check-window.data.service')
+const sortingService = require('../services/sorting-ui.service')
```

Comment:
that should be then:
`const sortingUIService = require('../services/sorting-ui.service')`